mypy/checkmember.py

```diff
@@ -913,6 +919,22 @@ def analyze_class_attribute_access(
             return apply_class_attr_hook(mx, hook, AnyType(TypeOfAny.special_form))
         return None
 
+    if (
+        isinstance(node.node, Var)
+        and not node.node.is_classvar
+        and not hook
```

Comment:
`and not hook` was required here, or this test started to fail:

https://github.com/python/mypy/blob/89469ac51aff1eefcd693d87d621577370ddedb4/test-data/unit/check-custom-plugin.test#L974-L990